README.md

```diff
@@ -101,6 +101,7 @@ Oh, we are ashamed and want to fix it ASAP! But before fixing a bug we need to r
 * and most importantly - a use-case that fails
 
 A minimal reproduce scenario using http://plnkr.co/ allows us to quickly confirm a bug (or point out coding problem) as well as confirm that we are fixing the right problem.
+The best part is that you don't need to create plunks from scratch - you can fork one from our [demo page](https://ng-bootstrap.github.io/#/components).
```

Comment:
Better to stay away from contractions in documentation - change `don't` to `do not`. It creates more formality.